packages/workspace/src/browser/workspace-delete-handler.ts

```diff
@@ -42,27 +38,25 @@ export class WorkspaceDeleteHandler implements UriCommandHandler<URI[]> {
     protected readonly workspaceService: WorkspaceService;
 
     /**
-     * Determine if the given command is visible
-     * @param uris
-     * @returns true if the given @param uris is not undefined, and does not contain a root directory
+     * Determine if the given delete command is visible.
+     *
+     * @returns `true` if uris is not `undefined` and does not contain a root directory
      */
     isVisible(uris: URI[]): boolean {
         return !!uris.length && !this.workspaceUtils.containsRootDirectory(uris);
     }
 
     /**
-     * Determine if the given command is enabled
-     * @param uris
-     * @returns true if the given @param uris is not undefined, and does not contain a root directory
+     * Determine if the given delete command is enabled.
+     *
+     * @returns `true` if uris is not `undefined` and does not contain a root directory
```

Comment:
I think it's not necessary to describe in that comment how the function you are documenting is implemented.  The purpose of the this comment is to say _what_ the function does (especially what the interface is), not _how_ it does it (unless there is a specific reason that callers should know that).

So here I would simply say:

    /**
     * Determine if the command is enabled.
     *
     * @returns `true` if the command is enabled.
     */

If you think it's necessary to talk about _how_ the functions determines whether the command is be enabled, then you can describe it like:

    /**
     * Determine if the command is enabled.
     * 
     * The command is considered enabled if some items are selected, but the selection
     * doesn't contain any root directory.
     *
     * @returns `true` if the command is enabled.
     */

But if you consider that a caller doesn't need to know how the function is implemented, then you can just move the detailed explanation to a comment inside the function.  This way, somebody reading the `isEnabled` function to try to understand how it works will have an explanation of what the intent of the code.

Finally, a good way to document the `uris` param might be

    @param uris URIs of selected resources.